spark/pyspark/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7.3-stretch
+
+ARG BAZEL_REPO="deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8"
+ARG BAZEL_RELEASE_KEY_URL=https://bazel.build/bazel-release.pub.gpg
+
+RUN apt-get update && \
```

Comment:
@ValentinMouret Best practice to put `&&` on the next line https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Below is a well-formed RUN instruction that demonstrates all the apt-get recommendations.
```
RUN apt-get update && apt-get install -y \
    aufs-tools \
    automake \
    build-essential \
    curl \
    dpkg-sig \
    libcap-dev \
    libsqlite3-dev \
    mercurial \
    reprepro \
    ruby1.9.1 \
    ruby1.9.1-dev \
    s3cmd=1.1.* \
 && rm -rf /var/lib/apt/lists/*
```

Reply:
```
RUN command \
<space>&& another command
```

Reply:
Yes, thank you :)
I figured it out rewrote it, commit coming.